packages/g/gamenetworkingsockets/xmake.lua

```diff
@@ -20,6 +20,7 @@ package("gamenetworkingsockets")
     end
 
     add_configs("webrtc", {description = "Enable p2p.", default = false, type = "boolean"})
+    add_requireconfs("**.abseil", {configs = {cxx_standard = "14"}})
```

Comment:
you should add `set_languages("c++17")` to https://github.com/xmake-io/xmake-repo/blob/f6cb475280bc1c40a51b9c330e04615e310b58e0/packages/g/gamenetworkingsockets/port/xmake.lua#L429